buildSrc/subprojects/configuration/src/main/kotlin/org/gradle/gradlebuild/BuildEnvironment.kt

```diff
@@ -11,7 +11,7 @@ object BuildEnvironment {
     const val BUILD_COMMIT_ID = "BUILD_COMMIT_ID"
 
     val isCiServer = CI_ENVIRONMENT_VARIABLE in System.getenv()
-    val isIntelliJIDEA by lazy { System.getProperty("idea.version") != null }
+    val isIntelliJIDEA by lazy { true }
```

Comment:
TODO: remove this once I'm happy with the spike, it's just a workaround for not being to run in the IDE